### PR TITLE
fix : fix config file not work for cz

### PR DESCRIPTION
**Pull request by wgjak47**
fix the config files like .cz didn't work when using `cz`

### CODE REVIEW

**Comment by Woile** [comment]
Hey thanks! Gonna tests and merge if everything goes ok :+1:

File: commitizen/cli.py [hunk 1]
```diff
@@ -78,7 +77,6 @@ def load_cfg():
     for cfg in configs:
         if not os.path.exists(config_file) and os.path.exists(cfg):
             config_file = cfg
-            break
```
Comment (Woile) [inline commitizen/cli.py]: This `break` is not needed. If we remove it, then the app will pick up the latest. 
The idea is to pick up the first one and continue.

**Comment by Woile** [comment]
Thanks, minor comment, once addressed it can be merged :+1:

File: commitizen/cli.py (hunk 1 above)
Comment (wgjak47) [reply commitizen/cli.py]: But there is another break after `settings.update(dict(config.items("commitizen")))`, and it ensures to pick up the first one and continue.

**Comment by Woile** [comment]
Can u rebase the PR and fix the conflicts? 
Thanks!

**Comment by wgjak47** [comment]
I will try to reopen a new PR... Looks like I am not a master of git rebase

**Pull request closed without merging**